.github/workflows/ci.yml

```diff
@@ -37,6 +37,14 @@ jobs:
         uses: actions/checkout@v2
       - name: "Run TypoScript lint"
         uses: TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint@v1
+  codestyle-check:
+    name: "Codestyle check"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      - name: "Run Copypaste Detector"
```

Comment:
This looks like a copy'n'paste error. ;-)

Reply:
that's true :) Thanks!